meta-openbmc-machines/meta-openpower/conf/machine/include/openpower.inc

```diff
@@ -4,6 +4,7 @@ OBMC_MACHINE_FEATURES += "\
         obmc-phosphor-sensor-mgmt \
         obmc-phosphor-flash-mgmt \
         obmc-phosphor-event-mgmt \
+        obmc-phosphor-user-mgmt \
```

Comment:
It took a few tries to get the package into the image. Removing the above line doesn't affect the image (user package is still included). Will refresh in a min. Thanks Brad for the explanation.